packages/core/src/Stores/notification-store.js

```diff
@@ -1536,6 +1539,23 @@ export default class NotificationStore extends BaseStore {
                 },
                 type: 'warning',
             },
+            additional_kyc_info: {
+                key: 'additional_kyc_info',
+                header: localize('Pending action required'),
```

Comment:
header is of type  JSX.Element. Lets use Localize

Reply:
Thanks @amina-deriv 
Updated